src/core/instance.ts

```diff
@@ -140,6 +140,34 @@ export default class Instance {
     }
   }
 
+  /**
+   * Auto generate translate X or Y position
+   * @param syntax styles syntax
+   */
+   transformTranslate(syntax: string) {
+    if (/(-translate|translate)-(x|y)-([0-9]{1,3}$)/.test(syntax)) {
+      let _updatedObject: object = {}
+      const extractTranslate: string[] = syntax.split("-")
+      const isNegative: boolean = syntax.includes("-translate")
+      const lastIndex: number = extractTranslate.length - 1
+      const value: number = isNegative ? Number(-extractTranslate[lastIndex]) : Number(extractTranslate[lastIndex])
+
+      if (extractTranslate.includes("x")) {
+        _updatedObject = {
+          transform: [{ translateX: value }]
+        }
+      }
+
+      if (extractTranslate.includes("y")) {
+        _updatedObject = {
+          transform: [{ translateY: value }]
+        }
+      }
+
+      this.updateObject(_updatedObject)
```

Comment:
I think it's better to return directly instead of creating a temporary variable to store object style

```jsx
if (extractTranslate.includes("x")) {
  this.updateObject({
    transform: [{ translateX: value }]
  })
}

if (extractTranslate.includes("y")) {
  this.updateObject({
    transform: [{ translateY: value }]
  })
}
```